dartagnan/src/main/java/com/dat3m/dartagnan/verification/solving/RefinementSolver.java

```diff
@@ -442,16 +443,22 @@ private static CharSequence generateCoverageReport(Set<Event> coveredEvents, Pro
         final Set<Integer> coveredOIds = new HashSet<>();
         final Set<Integer> coveredBranches = new HashSet<>();
         for (Event e : coveredEvents) {
-            if (e.getCLine() > 0) {
+            if (e.getCLine() > 0 && e.getOId() != -1) {
```

Comment:
Maybe we should add `hasOId`, `hasCId`, `hasUid` and `hasCSourceInformation` helper functions to `Event` which do the check against `-1`, so we can get rid of the magic constants.
Just the fact that we have the check `getCline() > 0` but `getOId() != -1` shows how inconsistently we query for the (non)-existence of metadata.